config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "IMAGE_DIR": "{replace_me}",
```

Comment:
Wenn die zu ersetzen sind, dann sollten die eigentlich aus der Umgebung kommen. Sonst haben wir mit jeder Änderung in der Config ein neues Release, was notwendig ist.